Code/GraphMol/Wrap/rough_test.py

```diff
@@ -4972,6 +4964,42 @@ def testGitHub1985(self):
     except:
        self.fail("[C@] caused an exception when roundtripping smarts")
 
+  def testGetEnhancedStereo(self):
+
+    rdbase = os.environ['RDBASE']
+    filename = os.path.join(rdbase, 'Code/GraphMol/FileParsers/test_data/two_centers_or.mol')
+    suppl = Chem.SDMolSupplier(filename)
```

Comment:
If you want to read a single molecule from a mol file (or sdf for that matter), it's easier to just use: `m = Chem.MolFromMolFile(filename)`

Reply:
nice